src/main/java/thePackmaster/cards/goddessofexplosionspack/AtomicPiledriver.java

```diff
@@ -0,0 +1,40 @@
+package thePackmaster.cards.goddessofexplosionspack;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.BorderFlashEffect;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class AtomicPiledriver extends AbstractGoddessOfExplosionsCard {
+    public final static String ID = makeID("AtomicPiledriver");
+
+    private static final int DAMAGE = 24;
+    private static final int DAMAGE_UP = 6;
+
+    public AtomicPiledriver() {
+        super(ID, 2, CardType.ATTACK, CardRarity.SPECIAL, CardTarget.ENEMY);
+        baseDamage = DAMAGE;
+        selfRetain = true;
+        exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        AbstractDungeon.effectsQueue.add(new BorderFlashEffect(Color.RED, true));
+        CardCrawlGame.sound.play("STANCE_ENTER_WRATH");
+        Wiz.atb(new DamageAction(m, new DamageInfo(p, damage, damageTypeForTurn), AbstractGameAction.AttackEffect.BLUNT_HEAVY));
```

Comment:
Could also use the explosion vfx after the blunt heav. The one that the explosive potion uses